### PR TITLE
[FUSEQE-6043] Deploy operator from generated resources

### PR DESCRIPTION
**Pull request by asmigala**
Partial implementation of FUSEQE-6043 (only the operator deployment implemented, still needs a way to specify the CR)

//test: `@smoke`

### CODE REVIEW

File: utilities/src/main/java/io/syndesis/qe/templates/SyndesisTemplate.java [hunk 1]
```diff
@@ -181,22 +182,44 @@ private static void deployCrd() {
...
-            "-n", TestConfiguration.openShiftNamespace(),
-            "-f", TestConfiguration.syndesisOperatorUrl()
+
+        log.info("Generating resources using operator image " + TestConfiguration.syndesisOperatorUrl());
+
+        ProcessBuilder pb = new ProcessBuilder("docker",
+            "run",
+            "--entrypoint",
+            "operator",
+            TestConfiguration.syndesisOperatorUrl(),
```
Comment (asmigala) [inline utilities/src/main/java/io/syndesis/qe/templates/SyndesisTemplate.java]: I guess I should create a new config prop rather than reuse this one

**Comment by syndesisqe** [comment]
Following scenarios failed:
```
Connection - CRUD | Validate all credentials 
Connection - CRUD | Create and delete 
Connection - CRUD | Kebab menu 
Connection - CRUD | Kebab menu edit and view 
Integration - DB to DB | Read and update operations 
```

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

File: utilities/src/main/java/io/syndesis/qe/templates/SyndesisTemplate.java (hunk 1 above)
Comment (avano) [reply utilities/src/main/java/io/syndesis/qe/templates/SyndesisTemplate.java]: :D maybe it is a good time to solve https://issues.jboss.org/browse/FUSEQE-3699

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by asmigala** [comment]
@avano do you want me to squash to a single commit? Other than that, should be good to go

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Comment by avano** [comment]
> @avano do you want me to squash to a single commit?

yes please

**Comment by asmigala** [comment]
@avano done

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Pull request merged**